changelog/12167.trivial.rst

```diff
@@ -0,0 +1 @@
+cache: create supporting files (``CACHEDIR.TAG``, ``.gitignore``, etc.) in the cache directory before writing cache data, reducing the probability of those files being lost.
```

Comment:
This probably needs to be updated now right?

Reply:
Yes, will do.